src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/DependenciesTreeSearchProvider.cs

```diff
@@ -97,15 +97,10 @@ Task SearchSolutionAsync()
             async Task SearchProjectAsync(UnconfiguredProject unconfiguredProject)
             {
                 IUnconfiguredProjectVsServices? projectVsServices = unconfiguredProject.Services.ExportProvider.GetExportedValue<IUnconfiguredProjectVsServices>();
-                IActiveConfigurationGroupService? configurationGroupService = unconfiguredProject.Services.ExportProvider.GetExportedValue<IActiveConfigurationGroupService>();
                 IProjectTree? dependenciesNode = projectVsServices?.ProjectTree.CurrentTree?.FindChildWithFlags(DependencyTreeFlags.DependenciesRootNode);
 
-                if (projectVsServices != null &&
-                    dependenciesNode != null &&
-                    configurationGroupService is IActiveConfigurationGroupService2 activeConfigurationGroupService)
+                if (projectVsServices != null && dependenciesNode != null)
                 {
-                    IConfigurationGroup<ConfiguredProject> configuredProjects = await activeConfigurationGroupService.GetActiveLoadedConfiguredProjectGroupAsync();
-
                     var projectContext = new DependenciesTreeProjectSearchContext(context, unconfiguredProject, dependenciesNode, _hierarchyItemManager, projectVsServices, _relationProvider);
```

Comment:
Where are the configured projects filtered later? I couldn't find any callers of http://index/?leftProject=Microsoft.VisualStudio.ProjectSystem.Managed.VS&leftSymbol=rrfjjefjnhg9&file=ProjectSystem%5CVS%5CTree%5CDependencies%5CAttachedCollections%5CIDependenciesTreeProjectSearchContext.cs.

Reply:
Here's an implementation of a search provider in the NuGet PR: https://github.com/NuGet/NuGet.Client/pull/3392/files#diff-fe1415c090d7eae9a266c217f9040c9dR75